src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/HtmlFormattingPass.cs

```diff
@@ -125,6 +125,48 @@ private static List<TextChange> AdjustRazorIndentation(FormattingContext context
 
                 if (context.Indentations[i].StartsInCSharpContext)
                 {
+                    // Normally we don't do HTML things in C# contexts but there is one
+                    // edge case when including render fragments in a C# code block, eg:
+                    //
+                    // @code {
+                    //      void Foo()
+                    //      {
+                    //          Render(@<SurveyPrompt />);
```

Comment:
Is it really this an not `Render(@"<SurveyPrompt />");`?

Reply:
Yep, it's really this.